docs/webiny-apps/headless-cms/features/roles-permissions.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I don't think this was included in `sidebars.js` I don't see this in the dropdown menu
![image](https://user-images.githubusercontent.com/9532882/83657784-513bc780-a58f-11ea-8cd9-9ef26b0cc9d5.png)